nujo/utils/data/dataset.py

```diff
@@ -1,8 +1,8 @@
 from numpy import array
 
-from nujo.utils.data.nujo_dir import HOME_DIR
+from nujo._typing import Union
 from nujo.utils.data.dataset_iterator import DatasetIterator
-from nujo.utils.data.dataset_loader import DatasetLoader
+from nujo.utils.data.dataset_loader import DatasetLoader, ndarray
```

Comment:
Don't import third-party things like that (talking about `ndarray`). Import them from their packages.
A.k.a. from numpy.